benchopt/plotting/html/templates/result.mako.html

```diff
@@ -55,7 +55,19 @@ <h1 class="ml11">
                             </div>
                         </div>
                         <div class="mt-8 px-8">
-                            <label for="objective_column" class="block text-sm font-medium text-white">Objective column</label>
+                            <div class="flex flex-row items-center justify-between text-sm font-medium text-white">
+                                <label for="objective_column" class="block">Objective column</label>
+                                <!--don't show description unless it is provided -->
+                                % if result['metadata']['obj_description'] != "":
+                                    <span class="objective-description-trigger" style="cursor: pointer;">
+                                        <span style="margin-right: 0.5em;">(description)</span>
```

Comment:
nitpick: remove the parenthesis:
```suggestion
                                        <span style="margin-right: 0.5em;">description</span>
```